src/app/Main.java

```diff
@@ -1,4 +1,4 @@
 package app;
 
 public class Main {
-}
+}
```

Comment:
just so GitHub doesn't get mad, make sure to add an extra line after class method